src/ontology/mondo.Makefile

```diff
@@ -264,7 +264,38 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts update
+## - main pipeline
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+tmp/bad-medgen-xrefs.txt: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/bad_medgen_xrefs_collate.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
+
+# Creates a command that, when run, will create mondo-edit.obo.tmp
+tmp/bad-medgen-xrefs-grep-command.sh: tmp/bad-medgen-xrefs.txt
+	python ../scripts/bad_medgen_xrefs_grep_command.py -i tmp/bad-medgen-xrefs.txt -t mondo-edit.obo -o $@
+
+# Creates mondo-edit.obo.tmp by using command
+mondo-edit.obo.tmp: tmp/bad-medgen-xrefs-grep-command.sh
+	@read -r grep_cmd < tmp/bad-medgen-xrefs-grep-command.sh; \
+    eval "$$grep_cmd"
```

Comment:
### How I constructed/ran `grep -v`
1. XLSX -> Single .txt of IDs
2. IDs -> construct grep string
3. Run `eval` on grep string

### Eval
I know it's not usually safe to run `eval`, but this is a temporary thing and I'm thinking I'll do a final commit where I remove some/all of these scripts.

Some alternatives I could have done for this: (i) create the grep command via some ShellScript and have make run it, (b) Run the grep command from Python using `subprocess.

Reply:
That's better! Honestly IDK if eval is any less safer, but your recommendation is just more straightforward / simpler. IDK why I didn't think of that. Changed~